src/components/Editor/HighlightLine.js

```diff
@@ -46,60 +46,32 @@ function isDocumentReady(selectedSource, selectedLocation) {
 }
 
 export class HighlightLine extends Component<Props> {
-  isStepping: boolean = false;
   previousEditorLine: ?number = null;
 
   shouldComponentUpdate(nextProps: Props) {
-    const { selectedLocation, selectedSource } = nextProps;
-    return this.shouldSetHighlightLine(selectedLocation, selectedSource);
-  }
-
-  shouldSetHighlightLine(selectedLocation: Location, selectedSource: Source) {
-    const { sourceId, line } = selectedLocation;
-    const editorLine = toEditorLine(sourceId, line);
-
-    if (!isDocumentReady(selectedSource, selectedLocation)) {
-      return false;
-    }
-
-    if (this.isStepping && editorLine === this.previousEditorLine) {
-      return false;
-    }
-
-    return true;
+    const { selectedSource, selectedLocation } = nextProps;
+    return isDocumentReady(selectedSource, selectedLocation);
   }
 
   componentDidUpdate(prevProps: Props) {
-    const {
-      pauseCommand,
-      selectedLocation,
-      selectedFrame,
-      selectedSource
-    } = this.props;
```

Comment:
I moved these to `setHighlightLine` since all this did was pass them on to that function anyway.